From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Wed, 27 Sep 2023 13:46:08 +0200
Subject: [PATCH] arm: dts: imx8mp-skov: Simplify SD partition definition to 1
 cell

Update the partition definitions for SD card in imx8mp-skov to use
single cell for address and size, streamlining the format.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm/dts/imx8mp-skov.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/dts/imx8mp-skov.dts b/arch/arm/dts/imx8mp-skov.dts
index 1d4738dbd7b1..b94b9dd51d6d 100644
--- a/arch/arm/dts/imx8mp-skov.dts
+++ b/arch/arm/dts/imx8mp-skov.dts
@@ -88,17 +88,17 @@ reg_usdhc2_vmmc: regulator-usdhc2 {
 &usdhc2 {
 	partitions {
 		compatible = "fixed-partitions";
-		#address-cells = <2>;
-		#size-cells = <2>;
+		#address-cells = <1>;
+		#size-cells = <1>;
 
 		partition@0 {
 			label = "barebox";
-			reg = <0x0 0x0 0x0 0x100000>;
+			reg = <0x0 0x100000>;
 		};
 
-		env_sd: partition@e0000 {
+		env_sd: partition@100000 {
 			label = "barebox-environment";
-			reg = <0x0 0x100000 0x0 0x100000>;
+			reg = <0x100000 0x100000>;
 		};
 	};
 };
